space-concordia-robotics/robotics-prototype
# [#143] added wifi image and duplicated ping rover function, working on modifiyng according

# Assignee Section

## Description
Added wifi image in top left of gui in the navbar.html template, which inherits over all other html pages. In addition, made a duplicate of the ping rover function. Need to fix that.

[If applicable, include a list of main points]
- Example bug fixed.
- Added example function with the responsibility of example.

### Steps for Testing
[Write all the steps necessary to test the feature(s)]
1) Run ./app.py in the basestation folder
2) Type localhost:5000 on an internet page
3) The GUI will load and a wifi icon in the top left will appear

closes #143

The approval from all software team leads is necessary before merging.

# Reviewer Section

Aside from local testing and the General Integration Test it is implied that static analysis should be included in the verification process.


For Pull Requests that do not include code changes, it is not required to perform the tests above.
